tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "./.nuxt/tsconfig.json",
+  "extends": "./docs/.nuxt/tsconfig.json",
```

Comment:
Why change this?

Reply:
Did you run `pnpm run dev:prepare`? https://ui.nuxt.com/getting-started/contributing#_2-local-development-setup

Reply:
no :sweat_smile: 

I remove this change